x/dex/keeper/utils/wasm.go

```diff
@@ -2,11 +2,44 @@ package utils
 
 import (
 	"encoding/json"
+	"time"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/sei-protocol/sei-chain/utils/metrics"
 	"github.com/sei-protocol/sei-chain/x/dex/keeper"
+	dextypeswasm "github.com/sei-protocol/sei-chain/x/dex/types/wasm"
 )
 
+func getMsgType(msg interface{}) string {
+	switch msg.(type) {
+	case dextypeswasm.SudoNewBlockMsg:
+		return "new_block"
+	case dextypeswasm.SudoFinalizeBlockMsg:
+		return "finalize_block"
+	case *dextypeswasm.SudoFinalizeBlockMsg:
+		return "finalize_block"
+	case dextypeswasm.SudoSettlementMsg:
+		return "settlement"
+	case dextypeswasm.SudoOrderPlacementMsg:
+		return "bulk_order_placements"
+	case dextypeswasm.SudoOrderCancellationMsg:
+		return "bulk_order_cancellations"
+	case dextypeswasm.SudoLiquidationMsg:
+		return "liquidation"
+	default:
+		return "unknown"
```

Comment:
should cover all the cases, but if we add more msg types in the future we'd need to update this 

![image](https://user-images.githubusercontent.com/18161326/187045025-a70407f8-0f73-43d7-8d54-9ef0cd5cfdf3.png)